tests/apollo/test_skvbc_view_change.py

```diff
@@ -158,7 +158,11 @@ async def test_single_vc_primary_isolated(self, bft_network, tracker):
                 err_msg="Make sure view change has been triggered."
             )
 
-            await tracker.tracked_read_your_writes()
+            with trio.fail_after(seconds=60):
+                while True:
+                    with trio.move_on_after(seconds=5):
```

Comment:
Might be, make this less than the client timeout of 5 seconds, just to be sure it will be the first to expire?